xml/cl.xml

```diff
@@ -1188,6 +1188,7 @@ has no formal schema defined.
     <extension number="49" name="cl_intel_planar_yuv"/>
     <extension number="50" name="cl_intel_device_side_avc_motion_estimation"/>
     <extension number="51" name="cl_intel_media_block_io"/>
+    <extension number="52" name="cl_amd_planar_yuv"/>
     <!-- NOTE: extension numbers are now assigned from
```

Comment:
Note the comment - please follow the "Adding Extension Specifications" process in the toplevel README.md.